polytracker/include/polytracker/passes/utils.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2022-present, Trail of Bits, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed in accordance with the terms specified in
+ * the LICENSE file found in the root directory of this source tree.
+ */
+
+#include <string>
+#include <unordered_set>
+#include <vector>
+
+namespace polytracker {
+using str_set_t = std::unordered_set<std::string>;
+using str_vec_t = std::vector<std::string>;
```

Comment:
> Only used for this particular function.
Yes, but they're used in the prototype so I think the names need to be exported so that users of `readIgnoreLists` can use them. Is there a better way to do this?

Reply:
This is a nitpick, really. If we don't use the names elsewhere we could just 
```c++
std::unordered_set<std::string> readIgnoreLists(std::vector<std::string>...
```
up to you.

Reply:
I'm inclined to leave them there. They're useful abbreviations people are likely to use elsewhere. In which case it might be a good idea to make it visible wider than in the `passes` folder.